### DISCUSSION TITLE
Custom HTML in thumbs?

### DISCUSSION THREAD

**Discussion by devjamin** [fancyapps/ui#688] [General]
Hi,

Is it possible to add some custom HTML in the thumbs when i call appendSlide? I have some slides in which I want to display a icon on top of the thumb as they are third-party dynamically loaded images.

**Comment by fancyapps** [comment]
Hi,

Here is one possible solution: 

```js
Fancybox.bind("[data-fancybox]", {
  Thumbs : {
    Carousel : {
      on: {
        "*": (carousel, eventName, slide) => {
          if (eventName === "createSlide") {
            let div = document.createElement("div");
            div.innerHTML = 'Hello';

            slide.el.append(div);
          }
        },
      },
    }
  }
});  
```

https://jsfiddle.net/u1kby75r/

**Reply by devjamin** [reply]
Thank you for the answer and demo! This is exactly what I was looking for :)
